src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.model;
+
+public class IsoscelesTrapezoid extends Color {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
```

Comment:
The formula for the area of an isosceles trapezoid is incorrect. It should be 0.5 * (base1 + base2) * height.